frontend/packages/process-editor/src/components/ConfigPanel/EditDataTypes/EditDataTypes.tsx

```diff
@@ -52,10 +52,11 @@ export const EditDataTypes = ({
         />
       ) : (
         <StudioProperty.Button
-          aria-label={t('process_editor.configuration_panel_set_data_model')}
+          aria-label={t('process_editor.configuration_panel_set_data_model', {
+            dataModelName: existingDataTypeForTask,
```

Comment:
Changed the aria-label from `datamodell:` to `datamodell: name-on-model`. This makes the aria-label more correct and we can wait for correct values in e2e-tests as well. 😊